source/includes/_api-reference.md

```diff
@@ -85,16 +85,32 @@ Response Code | Meaning
 ---------- | -------
 200 | Success -- Enjoy your data!
 201 | Created -- We created the record!
-204 | No Content -- Your request was successful, but we don't have anything more to tell you.
+204 | No Content -- Your request was successful, but we don't have anything more to tell you
 400 | Bad Request -- Something is wrong with your request
 401 | Unauthorized -- Your credentials are wrong
 403 | Forbidden -- You do not have permission to access the resource
 404 | Not Found -- The object or navigation property you are trying to access doesn't exist
 405 | Method Not Allowed -- We won't let you do that
+429 | Too Many Requests -- You have sent too many requests and have to wait till you can send more
 500 | Internal Server Error -- We had a problem with our server. Try again later
 501 | Not Implemented -- What you are trying to do doesn't work...yet.
 
 ## Versioning
 
 When we make backwards-incompatible changes to the API, we release new versions. The current version is **v2** and can be determined with our API base path **/api/v2/**. Read our [Intelex platform release notes](https://community.intelex.com/library/knowledgebase/release-notes) to see our API changelog.
 
+## API Rate Limiting / Quotas
+To ensure Intelex Platform is reliable, and has expected performance for all users, we limit the number of API calls an API user(or application) can make within a given time period. If the limit is exceeded, API user may be throttled and such requests will fail (status code 429).
+
+### Limit
+Intelex Platform API Rate limit is **2 requests per second**
```

Comment:
This seems to be too low for Intelex Mobile. When creating/editing records, we can fetch auxiliary records. When synching for offline use, we are fetching as much as possible to reduce the total sync time.
We would have to implement controls to adhere to these quotas. @ahmedtausif would you like to add more here?

Reply:
API rate limit will apply ONLY to the external integrations. Initially to requests that use ApiKey authentication.